scipy/stats/_stats_py.py

```diff
@@ -1861,10 +1861,11 @@ def normaltest(a, axis=0, nan_policy='propagate'):
     return NormaltestResult(k2, distributions.chi2.sf(k2, 2))
 
 
-Jarque_beraResult = namedtuple('Jarque_beraResult', ('statistic', 'pvalue'))
+JarqueBeraResult = namedtuple('JarqueBeraResult', ('statistic', 'pvalue'))
```

Comment:
We could, but the attributes can already be accessed by name, so I didn't bother. I can if you'd like.

Reply:
I would since we are at it. For consistency. Otherwise it's hard for newcomers to know what to use when adding a new test etc. And if one day we deprecate the unpacking behavior, we better have everything using a common architecture.

Reply:
Done.